source/operations/install-deploy-manage/deploy-operator-helm.rst

```diff
@@ -0,0 +1,316 @@
+                  lastUpdateTime: "2023-05-11T14:57:44Z"
+                  message: ReplicaSet "minio-operator-674cf5cf78" is progressing.
+                  reason: ReplicaSetUpdated
+                  status: "True"
+                  type: Progressing
+              observedGeneration: 1
+              replicas: 2
+              unavailableReplicas: 2
+              updatedReplicas: 2
+		     
+      ``console-secret.yaml``:
+
+      Create a ``console-secret.yaml`` file with the following contents:
+
+      .. code-block:: shell
+         :class: copyable
+
+         apiVersion: v1
+         kind: Secret
+         metadata:
+           name: console-sa-secret
+           namespace: minio-operator
+           annotations:
+             kubernetes.io/service-account.name: console-sa
+         type: kubernetes.io/service-account-token
+
+   #. ``kubectl apply`` the YAML files to apply the configuration to your deployment:
+
+      .. code-block:: shell
+         :class: copyable
+
+         kubectl apply -f service.yaml
+         kubectl apply -f operator.yaml
+         kubectl apply -f console-secret.yaml
+
+#. Connect to the Operator Console
+
+   .. include:: /includes/common/common-install-operator-kubectl-validate-open-console.rst
+
+
+Deploy a Tenant
+~~~~~~~~~~~~~~~
+
+You can deploy a MinIO tenant using either the :ref:`Operator Console <minio-operator-console>` or Helm.
+To deploy a tenant with the Console, see :ref:`Deploy and Manage MinIO Tenants <minio-installation>`.
+
+To deploy a tenant with Helm:
+
+#. The following Helm command creates a MinIO tenant:
+
+   .. code-block:: shell
+      :class: copyable
+
+      helm install \
+      --namespace tenant-ns \
+      --create-namespace \
+      tenant-ns tenant-|operator-version-stable|.tgz
+
+#. Expose the tenant Console port
+
+   Use the ``kubectl port-forward`` command to temporarily forward traffic from the MinIO pod to the local machine:
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl port-forward pod/minio 9000 9090 -n minio-dev
+   
+   The command forwards the pod ports ``9000`` and ``9090`` to the matching port on the local machine while active in the shell.
+   The ``kubectl port-forward`` command only functions while active in the shell session.
+   Terminating the session closes the ports on the local machine.
+
+   .. note::
+      
+      The following steps of this procedure assume an active ``kubectl port-forward`` command.
+
+      To configure long term access to the pod, configure :kube-docs:`Ingress <concepts/services-networking/ingress/>` or similar network control components within Kubernetes to route traffic to and from the pod. Configuring Ingress is out of the scope for this documentation.
+
+#. Login to the MinIO Console
+
+   Access the tenant :ref:`minio-console` by opening a browser on the local machine and navigating to ``http://127.0.0.1:9090``.
+   Log in to the Console with the credentials ``myminio | minio123``.
```

Comment:
These lines are also (condensed) from `includes/k8s/quickstart.rst` is there something better elsewhere? `myminio | minio123` appears to be default credentials, but only existing mention is in a spark/hadoop integrations page. I'm not clear why this isn't minioadmin/minioadmin.

Reply:
After this in the original steps from @cniackz there was another `kubectl port-forward` to enable `mc`. I never got it to work, recommendations appreciated. It would be good to include.

(Also I need to test all this once more to confirm nothing broke in translation to rst.)

Reply:
The default access key and secret values are defined in this file in the Helm folder:
https://github.com/minio/operator/blob/a524d928933034e4dfb20437568081a0b7826ab3/helm/tenant/values.yaml#L6

Reply:
Aha! I presume you could update that if you knew about it. Like maybe if the docs mentioned it. 😄

Reply:
@cniackz what is important to know about modifying the contents of our Helm charts before using? Is that a normal thing to do?

Reply:
Added a brief mention about customization. I have more detail for a future update.